src/api-ms-win-core-synch-l1-2-0/main.cpp

```diff
@@ -139,11 +139,18 @@ extern "C" BOOL WINAPI WaitOnAddress(_In_reads_bytes_(AddressSize) volatile VOID
 
         if (!SleepConditionVariableSRW(&context.cv, &entry.lock, dwMilliseconds, 0))
         {
+#ifndef NDEBUG

```

Comment:
We should make sure we define this!

Reply:
As far as I know this is defined in `common.build.pre.props`.
Unlike `_DEBUG` is `NDEBUG` actually an official definition in the C language (probably in C++ as well).

Reply:
oh, i did not know that! cool